flycheck.el

```diff
@@ -173,6 +173,7 @@ attention to case differences."
     coq
     css-csslint
     css-stylelint
+    cuda
```

Comment:
The usual naming for checkers is `<language>-<tool-name>`, unless there is only one credible checker in practice.  Is `nvcc` the only CUDA checker out there?

Reply:
Yes nvcc is the only CUDA compiler out there. CUDA is a proprietary language made by nvidia, so there aren't any open source alternatives.

OpenCL is the open source counterpart to CUDA. I might write a checker for this in future when I need one.